instagram-app/src/components/Login/Login.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+
+export const Login = props => {
+    return (
+        <div>
+            <form onSubmit={props.login}>
+                <input 
+                    type='text'
+                    name='name'
+                    value='Username'
+                    onChange={props.search}
+                />
+                <input
+                    type='text'
+                    name='name'
+                    value='Password'
+                    onChange={props.search}
+                />
+                <input
```

Comment:
For form submit buttons it is conventional to use a `<button>` element with `type="submit"`, no `value` needed.